pkg/investigations/mapping/mapping.go

```diff
@@ -15,25 +15,26 @@ import (
 func GetInvestigation(alertTitle string) *investigation.Investigation {
 	// We currently map to the alert by using the title, we should use the name in the alert note in the future.
 	// This currently isn't feasible yet, as CPD's alertmanager doesn't allow for the field to exist.
-
-	// We can't switch case here as it's strings.Contains.
-	if strings.Contains(alertTitle, "has gone missing") {
+	switch {
+	case strings.Contains(alertTitle, "has gone missing"):
 		return investigation.NewInvestigation(chgm.Investigate, "ClusterHasGoneMissing")
-	} else if strings.Contains(alertTitle, "ClusterProvisioningDelay -") {
+	case strings.Contains(alertTitle, "ClusterProvisioningDelay -"):
 		return investigation.NewInvestigation(cpd.Investigate, "ClusterProvisioningDelay")
 	}
 
-	// Return early if experimental features are not enabled
-	if strings.ToUpper(os.Getenv("CAD_EXPERIMENTAL_ENABLED")) != "TRUE" {
-		return nil
-	}
-
-	logging.Warn("Flag CAD_EXPERIMENTAL_ENABLED is set, experimental CAD investigations are enabled!")
+	if experimentalFeaturesEnabled() {
+		logging.Warn("Flag CAD_EXPERIMENTAL_ENABLED is set, experimental CAD investigations are enabled!")
 
-	// Experimental investigations go here
-	if strings.Contains(alertTitle, "ClusterMonitoringErrorBudgetBurnSRE") {
-		return investigation.NewInvestigation(clustermonitoringerrorbudgetburn.Investigate, "ClusterMonitoringErrorBudgetBurnSRE")
+		// We don't care this is a single case switch (gocritic), this should just be extensible
+		switch { //nolint:gocritic
+		case strings.Contains(alertTitle, "ClusterMonitoringErrorBudgetBurnSRE"):
```

Comment:
Instead of ignoring the linter here, can we add a "default" case that logs a message? This is probably helpful for the above statement as well.

Reply:
Maybe something like...

```
default:
		logging.Infof("No investigation exists for %s", alertTitle)
```

Reply:
I think that log makes sense, but rather than having it as default statement, it should be before the `return nil` to cover both `switch-cases`.  Then we still need the nolint statement, though. 

Let me know what you think about the latest push.